### PR TITLE
feature/R2

### PR DESCRIPTION
**Pull request by inverseswirl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,28 @@
...
+**3. External entities:**
+
+These represent any flow of information or data from any other external organisation, application programming interface or system. These usually represent the edges of the DFD diagrams.
+
+**4. Data flows:**
+
+The data flow lines represent the input and output and the flow of data through processes, external entities and data stores.
+
+![dfd](./dfd.png)
+
```
Comment (jarroddalefolino) [inline README.md]: We would want different DFDs for:

 * User management
 * Exercise list management
 * User recording sets, reps of exercises

I recommend for the exercise list doing a bulk import with a npm task

Comment (inverseswirl) [reply README.md]: ok Thanks, will work on that.

File: README.md [hunk 2]
```diff
@@ -32,7 +32,11 @@ These represent any flow of information or data from any other external organisa
 
 The data flow lines represent the input and output and the flow of data through processes, external entities and data stores.
 
-![dfd](./dfd.png)
+![dfd1](./dfd1.png)
+App logic dfd
+
+![dfd2](./dfd2.png)
+Exercise list dfd
```
Comment (inverseswirl) [inline README.md:39]: added new dfd

File: README.md (hunk 1 above)
Comment (inverseswirl) [reply README.md]: Ji and I had a discussion to go with single user role where user manages his own account with login and logout access only. The exercise list process ultimately merges with the main app data flow once the data is imported.

**Review by jarroddalefolino — APPROVED** [review]

**Pull request merged**